portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,62 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
+.siteBackground {
```

Comment:
i think CSS classes usually have the form .site-background instead of lower camel case